spring-integration-file/src/test/java/org/springframework/integration/file/config/FileOutboundChannelAdapterParserTests-context.xml

```diff
@@ -52,7 +57,21 @@
 			<bean class="org.springframework.integration.file.config.FileOutboundChannelAdapterParserTests$FooAdvice" />
 		</file:request-handler-advice-chain>
 	</file:outbound-channel-adapter>
+	
+	<file:outbound-channel-adapter id="adapterUsageWithAppendAndAppendNewLineTrue"
+	                               filename-generator-expression="@fooString"
+								   mode="APPEND"
+								   append-new-line="true"
+								   directory-expression="@barString">	
+	</file:outbound-channel-adapter>
 
+	<file:outbound-channel-adapter id="adapterUsageWithAppendAndAppendNewLineFalse"
+	                               filename-generator-expression="@fooString"
+								   mode="APPEND"
+								   append-new-line="false"
+								   directory-expression="@barString">	
+	</file:outbound-channel-adapter>
```

Comment:
BTW, the `<int-file:outboud-gateway>` should support this atribute as well. According to the `outboundFileBaseType` XSD type it is already here. But we should have at least one test for that component to be sure parser populates it correctly. No need those integration tests as you did for the `FileOutboundChannelAdapterParserTests`